packages/next/src/server/lib/revalidate.ts

```diff
@@ -13,8 +13,8 @@ export function formatRevalidate(revalidate: Revalidate): string {
   if (revalidate === 0) {
     return 'private, no-cache, no-store, max-age=0, must-revalidate'
   } else if (typeof revalidate === 'number') {
-    return `s-maxage=${revalidate}, stale-while-revalidate`
+    return `s-maxage=${revalidate}, stale-while-revalidate=${revalidate}`
```

Comment:
```suggestion
    return `s-maxage=${revalidate}, stale-while-revalidate=${CACHE_ONE_YEAR}`
```
I believe we want the stale-while-revalidate delta to always be at least a year as this is the amount of time a cache is allowed to keep the stale value and if a resource is failing to revalidate we don't want to evict the stale content just as fast as we want to revalidate the content.